tests/rs_sqla_test_utils/models.py

```diff
@@ -44,7 +44,7 @@ class ReflectionSortKey(Base):
     col2 = sa.Column(sa.Integer())
     __table_args__ = (
         {'redshift_diststyle': 'EVEN',
-         'redshift_sortkey': ('col1, col2')}
+         'redshift_sortkey': ('col1', 'col2')}
```

Comment:
I'd agree that this is a breaking change that should be included in the CHANGES.rst